src/drivers/tap_esc/tap_esc_common.h

```diff
@@ -51,5 +54,17 @@ namespace tap_esc_common
  ****************************************************************************/
 void select_responder(uint8_t sel);
 
+int initialise_uart(const char *const device, int &uart_fd);
+
+int deinitialise_uart(int &uart_fd);
+
+int enable_flow_control(int uart_fd, bool enabled);
+
+int send_packet(int uart_fd, EscPacket &packet, int responder);
```

Comment:
Agreed, it would make more sense.  However, `send_packet` will calculate and fill in the CRC checksum. That's why it cannot be `const` at this point. We could of course take the CRC calculation out of `send_packet`, but I'm not sure if that's more intuitive :)

Reply:
Ah I did not see that. It's fine then.